avm/src/vm/deploy.rs

```diff
@@ -0,0 +1,185 @@
+use hyper::{client::Client, Body, Request};
+use serde::{Deserialize, Serialize};
+use serde_json::{from_reader, from_str, json, Value};
+
+use std::error::Error;
+use std::fmt::Display;
+use std::fs::{File, read};
+use std::io::BufReader;
+use std::path::Path;
+
+use ascii_table::{AsciiTable, Column, Align};
+use base64;
+use tokio::runtime::Runtime;
+
+const URL: &str = "https://alan-deploy-prod.herokuapp.com";
```

Comment:
Should we CNAME this to `deploy.alantechnologies.com`?

Reply:
Another thing; should we shove a `/v1/` into the URL paths so we can change the APIs in the future without breaking older versions of `alan`. If we get upstreamed into something like Debian there's going to be significant usage of an earlier version of alan just like we ourselves were using an earlier version of Rust for a while since 1.41 is what Debian still ships.

Reply:
> Another thing; should we shove a `/v1/` into the URL paths so we can change the APIs in the future without breaking older versions of `alan`. If we get upstreamed into something like Debian there's going to be significant usage of an earlier version of alan just like we ourselves were using an earlier version of Rust for a while since 1.41 is what Debian still ships.

Great point! I decouple the CLI from the rest api internals as much as possible, but let's do this!